src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -36,7 +36,7 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<HttpRequestBodySizeMiddleware>();
             builder.UseMiddleware<SystemTraceMiddleware>();
             builder.UseMiddleware<HostnameFixupMiddleware>();
-            if (environment.IsLinuxConsumption())
+            if (environment.IsLinuxConsumption() || environment.IsKubernetesManagedHosting())
```

Comment:
So Kubernetes environment is also using the Assign specialization code that activates the delay feature [here](https://github.com/Azure/azure-functions-host/blob/f1c6016b8e3a0f6ad8f02eb04297badf118e7acb/src/WebJobs.Script.WebHost/Management/InstanceManager.cs#L150)?

Reply:
Correct.